plop-templates/example/README.md

```diff
@@ -0,0 +1,41 @@
+# -- PLOP TITLE HERE -- example
+
+This example shows how -------------------------
```

Comment:
Can we fill this too? I mean, so we don't have to change anything, that would be awesome!

Reply:
you mean to create another CLI plop input to ask for a long description when creating the example? So I prepopulate title with `-- PLOP TITLE HERE --` and I can do that but I believe it would make sense for developer to populate this long description after the example is created and those large dashes indicate that this should be manually populated.